README.md

```diff
@@ -6,13 +6,13 @@ Role Based Access Control (**`RBAC`**) gives you
 a human-friendly way of controlling access
 to specific data/features in your App(s).
 
-[![Build Status](https://img.shields.io/travis/com/dwyl/rbac/master.svg?style=flat-square)](https://travis-ci.com/dwyl/rbac)
+[![Build Status](https://img.shields.io/travis/dwyl/rbac/master.svg?style=flat-square)](https://travis-ci.org/dwyl/rbac)
```

Comment:
derp! 🤦  I wasn't paying close enough attention last night. 😫 

![image](https://user-images.githubusercontent.com/194400/141466282-ba7efcb3-3828-4b0f-8579-4e05d17c8da2.png)

new PR for the build badge: https://github.com/dwyl/rbac/pull/21

Reply:
sorry should have spot that too

Reply:
Not at all. it was a subtle thing and that's why we have badges to inform us. 😉